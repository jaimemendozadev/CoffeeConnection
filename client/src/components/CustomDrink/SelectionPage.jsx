import React, {Component} from 'react'

class SelectionPage extends Component {
  render () {
    return (
      <div>SelectionPage</div>
    )
  }
}

export default SelectionPage
