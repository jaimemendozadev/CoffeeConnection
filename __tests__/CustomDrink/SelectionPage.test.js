import React from 'react'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import {SelectionPage} from '../../client/src/components/CustomDrink/SelectionPage.jsx'

configure({ adapter: new Adapter() })

import {shallow} from 'enzyme'

const selectionPage = shallow(<SelectionPage />)

describe('SelectionPage', () => {
  it('renders properly', () => {
    expect(selectionPage).toMatchSnapshot();
  })
  
})