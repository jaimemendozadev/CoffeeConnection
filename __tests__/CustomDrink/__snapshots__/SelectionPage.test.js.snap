// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectionPage renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SelectionPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          What Drink Would You Like?
        </h1>,
        <button>
          Coffee
        </button>,
        <button>
          Tea
        </button>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "What Drink Would You Like?",
        },
        "ref": null,
        "rendered": "What Drink Would You Like?",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Coffee",
        },
        "ref": null,
        "rendered": "Coffee",
        "type": "button",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Tea",
        },
        "ref": null,
        "rendered": "Tea",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            What Drink Would You Like?
          </h1>,
          <button>
            Coffee
          </button>,
          <button>
            Tea
          </button>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "What Drink Would You Like?",
          },
          "ref": null,
          "rendered": "What Drink Would You Like?",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Coffee",
          },
          "ref": null,
          "rendered": "Coffee",
          "type": "button",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Tea",
          },
          "ref": null,
          "rendered": "Tea",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
